test/scheduler_test.rb

```diff
@@ -507,11 +507,13 @@
   end
 
   test 'procline contains app_name when present' do
+    Resque::Scheduler.expects(:env).at_least_once.returns(nil)
```

Comment:
What is going on with these?

Reply:
The Rails environment name is included in the procline string. Because we default the `RAILS_ENV` to `test` now, we have to unset the `env` here to avoid breaking these specs. (It used to be blank by default.)